### ISSUE TITLE
Accessibility swarm on the Section Label component

### ISSUE THREAD

**Issue by DenisHdz** [BBC-archive/psammead#558]
**Is your feature request related to a problem? Please describe.**
Carry out a two `devs` accessibility swarm for the `SectionLabel` component.
 
**Describe the solution you'd like**
Go through this [checklist](https://bbc-news.github.io/accessibility-news-and-you/accessibility-news-and-testers) and fix any issue we might face.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

**Comment by DenisHdz** [comment]
After carried out the a11y swarm we found out two issues with the separator line:
- It doesn’t meet color contrast: 2.2:1 ratio (UX is fine with this)
- It disappears in High Contrast mode

**Comment by andrew-nowak** [comment]
#605 raised to deal with the line disappearing in high contrast

**Comment by andrew-nowak** [comment]
We ran a mini swarm to confirm #605 will be fixed by #611 and it will! :tada: 
https://github.com/bbc/psammead/pull/611#issuecomment-501185402
